### PR TITLE
Release/snowplow web/0.5.1

### PR DESCRIPTION
**Pull request by emielver**
## Description & motivation
Bump snowplow-utils version to support both mobile and web models at the same time

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ If you find a bug, please report an issue on GitHub.
 
 # Copyright and license
 
-The snowplow-web package is Copyright 2021 Snowplow Analytics Ltd.
+The snowplow-web package is Copyright 2022 Snowplow Analytics Ltd.
```
Comment (bill-warner) [inline README.md:60]: I think this should be 2021-2022

**Review by bill-warner — APPROVED** [review]
Just 1 small change but otherwise LGTM. Good stuff.

File: README.md (hunk 1 above)
Comment (emielver) [reply README.md]: Thanks for the heads up, should be all good now!

**Comment by bill-warner** [comment]
Looks good.

**Pull request merged**